docs/designers-developers/faq.md

```diff
@@ -267,39 +267,39 @@ We plan to continue supporting existing meta boxes while providing new ways to e
 
 The main extension point we want to emphasize is creating new blocks. We are still working on how to extend the rest of the UI that is built in JS. We are tracking it here: [Issue #1352](https://github.com/WordPress/gutenberg/issues/1352)
 
-## Will Custom Post Types be supported?
+## Are Custom Post Types still supported?
 
 Indeed. There are multiple ways in which custom post types can leverage Gutenberg. The plan is to allow them to specify the blocks they support, as well as defining a default block for the post type. It's not currently the case, but if a post type disables the content field, the “advanced” section at the bottom would fill the page.
 
-## Will there be columns?
+## Does Gutenberg support columns?
 
 Yes, a columns block is available in Gutenberg.
 
-## Will there be nested blocks?
+## Does Gutenberg support nested blocks?
```

Comment:
The answer also needs to get updated as nested blocks are part of Gutenberg.

Reply:
I don't understand why my suggestion/answer is above the question.

Reply:
I already suggested this answer in a previous comment but it seems it didn't go where it was supposed to. Let me put it here again:

```suggestion
Yes, it is supported. You can have multiple levels of nesting – blocks within blocks within blocks.
```

Reply:
I see it now. Can you include it in the patch? I will merge it afterward.